models/issue.go

```diff
@@ -69,6 +71,13 @@ func init() {
 	issueTasksDonePat = regexp.MustCompile(issueTasksDoneRegexpStr)
 }
 
+// AfterLoad checks if the issue is overdue and sets the property
+func (issue *Issue) AfterLoad() {
+	if util.TimeStampNow() >= issue.DeadlineUnix {
+		issue.IsOverDue = true
```

Comment:
why not just change IsOverDue as a method of Issue?

Reply:
Good idea, I've changed it.